tests/test_base_retrieval_evaluation_factory.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
just make new directory for retrieval_evaluation test

Reply:
is it not good? hitchhiker guide book brought me here...?!

Reply:
ah. like our source_code folder
I got it.
I will do that.
mm (it means.. my mistake lol)